terraform/common.tf

```diff
@@ -31,11 +31,11 @@ variable "sample_app_image" {
 }
 
 variable "aoc_image_repo" {
-  default = "611364707713.dkr.ecr.us-west-2.amazonaws.com/aws/aws-otel-collector"
+  default = "public.ecr.aws/aws-observability/aws-otel-collector"
 }
 
 variable "aoc_version" {
-  default = "v0.4.0-382067259"
+  default = "v0.6.0"
```

Comment:
Wondering, why can't we make it `latest`?

Reply:
We probably can, I just did a specific version because previously a specific version was also used

Reply:
since the integration test is testing the latest build that not being released. `latest` won't work. It will be always override from the terraform input. We can leave it as is.

Reply:
The integration always should always test against on the newly built images that's why you see "v0.4.0-38xxxx". We always take it from terraform input in the workflow. So a fixed version number or `latest` doesnt' matter too much here.